docs/tutorials/gluon/learning_rate_finder.md

```diff
@@ -0,0 +1,322 @@
+lr = 0.05
+
+for iter_idx in range(500):
+    learner.iteration(lr=lr)
+    if ((iter_idx % 100) == 0):
+        print("Iteration: {}, Loss: {:.5g}".format(iter_idx, learner.iteration_loss.asscalar()))
+print("Final Loss: {:.5g}".format(learner.iteration_loss.asscalar()))
+```
+
+Iteration: 0, Loss: 2.785 <!--notebook-skip-line-->
+
+Iteration: 100, Loss: 1.6653 <!--notebook-skip-line-->
+
+Iteration: 200, Loss: 1.4891 <!--notebook-skip-line-->
+
+Iteration: 300, Loss: 1.0846 <!--notebook-skip-line-->
+
+Iteration: 400, Loss: 1.0633 <!--notebook-skip-line-->
+
+Final Loss: 1.1812 <!--notebook-skip-line-->
+
+
+We see a sizable drop in the loss from approx. 2.7 to 1.2.
+
+And now we have a baseline, let's see what happens when we train with a learning rate that's higher than advisable at 0.5.
+
+
+```python
+net = mx.gluon.model_zoo.vision.resnet18_v2(classes=10)
+learner = Learner(net=net, data_loader=data_loader, ctx=ctx)
+learner.net.load_params("net.params", ctx=ctx)
+lr = 0.5
+
+for iter_idx in range(500):
+    learner.iteration(lr=lr)
+    if ((iter_idx % 100) == 0):
+        print("Iteration: {}, Loss: {:.5g}".format(iter_idx, learner.iteration_loss.asscalar()))
+print("Final Loss: {:.5g}".format(learner.iteration_loss.asscalar()))
+```
+
+Iteration: 0, Loss: 2.6469 <!--notebook-skip-line-->
+
+Iteration: 100, Loss: 1.9666 <!--notebook-skip-line-->
+
+Iteration: 200, Loss: 1.6919 <!--notebook-skip-line-->
+
+Iteration: 300, Loss: 1.3643 <!--notebook-skip-line-->
+
+Iteration: 400, Loss: 1.4743 <!--notebook-skip-line-->
+
+Final Loss: 1.366 <!--notebook-skip-line-->
+
+
+We still observe a fall in the loss but aren't able to reach as low as before.
+
+And lastly, we see how the model trains with a more conservative learning rate of 0.005.
+
+
+```python
+net = mx.gluon.model_zoo.vision.resnet18_v2(classes=10)
+learner = Learner(net=net, data_loader=data_loader, ctx=ctx)
+learner.net.load_params("net.params", ctx=ctx)
+lr = 0.005
+
+for iter_idx in range(500):
+    learner.iteration(lr=lr)
+    if ((iter_idx % 100) == 0):
+        print("Iteration: {}, Loss: {:.5g}".format(iter_idx, learner.iteration_loss.asscalar()))
+print("Final Loss: {:.5g}".format(learner.iteration_loss.asscalar()))
+```
+
+Iteration: 0, Loss: 2.605 <!--notebook-skip-line-->
+
+Iteration: 100, Loss: 1.8621 <!--notebook-skip-line-->
+
+Iteration: 200, Loss: 1.6316 <!--notebook-skip-line-->
+
+Iteration: 300, Loss: 1.6295 <!--notebook-skip-line-->
+
+Iteration: 400, Loss: 1.4019 <!--notebook-skip-line-->
+
+Final Loss: 1.2919 <!--notebook-skip-line-->
+
+
+Although we get quite similar results to when we set the learning rate at 0.05 (because we're still in the region of falling loss on the Learning Rate Finder plot), we can still optimize our network faster using a slightly higher rate.
+
+## Wrap Up
+
+Give Learning Rate Finder a try on your current projects, and experiment with the different learning rate schedules found in this tutorial too.
```

Comment:
I went back to look for the "different learning rate schedules" and they not explicitly defined. Or maybe I missed that point, but I wasn't really able to find it.